src/styles/typography/index.md.njk

```diff
@@ -190,3 +190,7 @@ You can use the `govuk-section-break` classes on an `<hr>` element to create a t
 By default `govuk-section-break` is only visible by its margin. You can add the `govuk-section-break--visible` class to make it visible with a separator line.
 
 {{ example({group: "styles", item: "typography", example: "section-break", html: true, open: true, size: "m"}) }}
+
+## Our plans to improve this style
+
+In 2023, we’ll be reviewing and improving our typography standards to make sure they’re up-to-date and as accessible as possible for users and service teams. [Read a blog post about our plans](https://designnotes.blog.gov.uk/2022/12/12/making-the-gov-uk-frontend-typography-scale-more-accessible/).
```

Comment:
Original plan was to put this text under "Help improve this style", but I forgot that heading is handled by a site-wide template () or partial.